src/arm/overlays/BBAI-4D4C-00A1.dts

```diff
@@ -0,0 +1,173 @@
+/*
```

Comment:
Does this work on BeagleBone Black now? Can you rename as appropriate?

Reply:
Nope, this only works on BBAI!

Reply:
Please note what deltas we cannot resolve.